app/src/main/java/com/konkuk/medicarecall/ui/feature/home/model/HomeResponseDto.kt

```diff
@@ -1,4 +1,4 @@
-package com.konkuk.medicarecall.ui.home.model
+package com.konkuk.medicarecall.ui.feature.home.model
```

Comment:
여기도 데이터 패키지 안의 request 패키지 안으로 옮겨도 괜찮을 거 같아요
```suggestion
package com.konkuk.medicarecall.data.dto.response

Reply:
이 data 관련 리팩토링은 위에서 말씀드린 것처럼, 월요일 회의에서 논의한 뒤에 한 번에 반영하도록 하겠습니다!